documentation/community/connecting_with_UMA.md

```diff
@@ -0,0 +1,4 @@
+# Connect with UMA
+
+- Join our [Slack](https://join.slack.com/t/umaprotocol/shared_invite/zt-7mtxxds5-OIhE~q_WkwGCVNrq0~G~rg) 
```

Comment:
nit: please use this slack link: https://join.slack.com/t/umaprotocol/shared_invite/enQtNTk4MjQ4ODY0MDA1LTM4ODg0NGZhYWZkNjkzMDE4MjU0ZGFlYWQzZTFiZWFlZjI2NDE4OGI2NWY3OTdhYjYyZjg0MjAzMTgwODVhZTE

Reply:
Yes - my bad!